apps/chat/src/utils/app/publications.ts

```diff
@@ -22,9 +22,6 @@ import { getFolderIdFromEntityId, splitEntityId } from './folders';
 import { isRootId } from './id';
 import { EnumMapper } from './mappers';
 
-export const isItemPublic = (id: string) =>
-  id.split('/')[1] === PUBLIC_URL_PREFIX;
-
```

Comment:
Why did you remove good util-method istead of modifying it and use this logic directly in all places? DRY: don't repeat yourself and keep repeatable logic in one place
```
export const isItemPublic = (id: string, featureType: FeatureType) =>
  id.startsWith(
            getRootId({
              featureType,
              bucket: PUBLIC_URL_PREFIX,
            }));
```
but first version works without specifying of `featureType` and I like it more